book.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Class that represents a book
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Good job adding the getter and setter for `title` and `author`. Please also set the Getters and Setters for `@rentals`. users should be able to get and set the rental of the book.